### PR TITLE
core: 's/eris-ltd/monax/g'

### PR DESCRIPTION
**Pull request by zramsay**
- renames import paths for new organization name
- vendor will need to be updated once compilers & edb are updated
- command used: `git grep -l 'eris-ltd' | xargs sed -i 's/eris-ltd/monax/g'`

### CODE REVIEW

File: glide.lock [hunk 1]
```diff
@@ -55,13 +55,13 @@ imports:
   version: 47238d5aae8c0fefd518ef2bee46290909cf8263
 - name: github.com/eapache/queue
   version: 44cc805cf13205b55f69e14bcb69867d1ae92f98
-- name: github.com/eris-ltd/eris-compilers
+- name: github.com/monax/compilers
   version: 80a2287ac76b8426dcf6bbbfd1e164b52154ba82
```
Comment (zramsay) [inline glide.lock]: will be updated in a seperate PR fixing glide issues

File: config/paths.go [hunk 2]
```diff
@@ -14,9 +14,9 @@ import (
 var (
 	// Convenience directories.
 	GoPath            = os.Getenv("GOPATH")
-	ErisLtd           = filepath.Join(GoPath, "src", "github.com", "eris-ltd") // CSK: to deprecate
-	ErisGo            = filepath.Join(GoPath, "src", "github.com", "eris-ltd") // CSK: to keep
-	ErisGH            = "https://github.com/eris-ltd/"
+	ErisLtd           = filepath.Join(GoPath, "src", "github.com", "monax") // CSK: to deprecate
```
Comment (benjaminbollen) [inline config/paths.go]: Worth renaming these paths and variable names?

File: misc/release/release.sh [hunk 3]
```diff
@@ -174,15 +174,15 @@ prepare_gh() {
   if [[ "$1" == "pre" ]]
   then
     github-release release \
-      --user eris-ltd \
+      --user monax \
       --repo eris \
```
Comment (benjaminbollen) [inline misc/release/release.sh]: eris -> cli?

File: misc/release/release.sh [hunk 4]
```diff
@@ -174,15 +174,15 @@ prepare_gh() {
...
       --tag v${LATEST_TAG} \
       --name "Release of Version: ${LATEST_TAG}" \
       --description "${DESCRIPTION}" \
       --pre-release
   else
     github-release release \
-      --user eris-ltd \
+      --user monax \
       --repo eris \
```
Comment (benjaminbollen) [inline misc/release/release.sh]: eris -> cli ?

File: misc/release/release.sh [hunk 5]
```diff
@@ -200,7 +200,7 @@ release_gh() {
   do
     echo "Uploading: ${file}"
     github-release upload \
-      --user eris-ltd \
+      --user monax \
       --repo eris \
```
Comment (benjaminbollen) [inline misc/release/release.sh]: eris -> cli ?

File: config/paths.go (hunk 2 above)
Comment (zramsay) [inline config/paths.go]: when we change the name of the binary & `~/.eris` becomes new name

File: misc/release/release.sh (hunk 3 above)
Comment (zramsay) [inline misc/release/release.sh]: ++

File: misc/release/release.sh (hunk 4 above)
Comment (zramsay) [inline misc/release/release.sh]: ++

File: misc/release/release.sh (hunk 5 above)
Comment (zramsay) [inline misc/release/release.sh]: :+1:

**Pull request merged**